Presentation.Web/app/components/it-contract/it-contract-overview.controller.ts

```diff
@@ -1,971 +1,963 @@
+                            this.contractTemplateOptionViewModel.getOptionText(dataItem.ContractTemplate?.Id)))
+                        .withExcelOutput(dataItem => Helpers.ExcelExportHelper.renderString(
+                            this.contractTemplateOptionViewModel.getOptionText(dataItem.ContractTemplate?.Id))))
+                    .withColumn(builder =>
+                        builder
+                        .withDataSourceName(this.purchaseFormPropertyName)
+                        .withTitle("Indkøbsform")
+                        .withId("purchaseForm")
+                        .withFilteringOperation(Utility.KendoGrid.KendoGridColumnFiltering.FixedValueRange)
+                        .withFixedValueRange(
+                            Kitos.Helpers.KendoOverviewHelper.mapDataForKendoDropdown(
+                                this.purchaseFormOptionViewModel.options,
+                                true),
+                            false)
+                        .withRendering(dataItem => Helpers.RenderFieldsHelper.renderString(
+                            this.purchaseFormOptionViewModel.getOptionText(dataItem.PurchaseForm?.Id)))
+                        .withExcelOutput(dataItem => Helpers.ExcelExportHelper.renderString(
+                            this.purchaseFormOptionViewModel.getOptionText(dataItem.PurchaseForm?.Id))))
+                    .withColumn(builder =>
+                        builder
+                        .withDataSourceName(this.procurementStrategyPropertyName)
+                        .withTitle("Genanskaffelsesstrategi")
+                        .withId("procurementStrategy")
+                        .withFilteringOperation(Utility.KendoGrid.KendoGridColumnFiltering.FixedValueRange)
+                        .withFixedValueRange(
+                            Kitos.Helpers.KendoOverviewHelper.mapDataForKendoDropdown(
+                                this.procurementStrategyOptionViewModel.options,
+                                true),
+                            false)
+                        .withRendering(dataItem => Helpers.RenderFieldsHelper.renderString(
+                            this.procurementStrategyOptionViewModel.getOptionText(dataItem
+                                .ProcurementStrategy?.Id)))
+                        .withExcelOutput(dataItem => Helpers.ExcelExportHelper.renderString(
+                            this.procurementStrategyOptionViewModel.getOptionText(dataItem
+                                .ProcurementStrategy?.Id))))
+                    .withColumn(builder =>
+                        builder
+                        .withDataSourceName(this.procurementPlanYearPropertyName)
+                        .withTitle("Genanskaffelsesplan")
+                        .withId("procurementPlanYear")
+                        .withFilteringOperation(Utility.KendoGrid.KendoGridColumnFiltering.FixedValueRange)
+                        .withFixedValueRange($scope.procurements, false)
```

Comment:
ProcurementPlan filtering is the only part where description wasn't 100% clear to me (must've missed it when reading through everything). I'm not sure if the api method I created was a correct approach